inference/inference.py

```diff
@@ -322,6 +322,23 @@ def autoregressive_inference(params, ic, valid_data_full, model):
             }
           )
               
+    # inspect snapshot times at 5-days, 10-days.
+    # N.B. the predictions are 1-indexed, i.e. index 0 is the first prediction.
+    snapshot_timesteps = [(24 // 6 * k - 1, f"{k}-days") for k in [5, 10]]
+
+    # TODO(gideond) move these names to a higher-level to avoid potential bugs
+    metric_names = ['rmse', 'acc', 'global_mean_prediction', 'global_mean_target', 'global_mean_gradient_magnitude_prediction', 'global_mean_gradient_magnitude_target']
+    # All metrics has shape [metric_type, timestep, channel]
+    all_metrics = [valid_loss, acc, global_mean_pred, global_mean_target, gradient_magnitude_pred, gradient_magnitude_target]
+    all_metrics = np.array([m.cpu().numpy() for m in all_metrics])
+    for i in range(len(metric_names)):
+        for j, time_name in snapshot_timesteps:
+          for k in range(len(out_names)):
+            name = f'{metric_names[i]}/ic{ic}/{out_names[k]}/{time_name}'
```

Comment:
```suggestion
            name = f'{metric_names[i]}_{time_name}/ic{ic}/channel{k}-{out_names[k]}'
```

Given how wandb organizes metrics, I prefer the channel to be the last. IMO this is handled most cleanly by saying we've defined a new metric called "RMSE_5-days". And I'd rather keep the channel # for now to be consistent with how the time-dependent metrics are logged.